cmd/net.go

```diff
@@ -77,18 +77,58 @@ func getHostIP4(host string) (ipList set.StringSet, err error) {
 	return ipList, err
 }
 
+// byLastOctetValue implements sort.Interface used in sorting a list
+// of ip address by their last octet value.
+type byLastOctetValue []net.IP
```

Comment:
> All we want is a highest octet on the system to be the first in the element not just by string comparing and moving localhost to the end.

Personally the highest octet is printed as first doesn't make any value to me and I see its just a matter of choice.  However I don't have any issue with this algo.